examples/04.MQTT/mqtt.cc

```diff
@@ -38,8 +38,10 @@ DECLARE_AND_DEFINE_CONNECTION_CAPABILITY(MosquittoOrgMQTT,
 
 DECLARE_AND_DEFINE_ALLOCATOR_CAPABILITY(mqttTestMalloc, 32 * 1024);
 
-const char *testTopic   = "cherries";
-const int   testPayload = 42;
+const char   testTopic[]     = "cherries";
+const size_t testTopicSize   = sizeof(testTopic) - 1;
+const int    testPayload     = 42;
+const size_t testPayloadSize = sizeof(testPayload) - 1;
```

Comment:
You can avoid all of the `-1` calculations here using `constexpr std::string_view` and `.size()`.